crates/edr_provider/src/requests/methods.rs

```diff
@@ -29,10 +27,10 @@ mod optional_block_spec {
 #[serde(tag = "method", content = "params")]
 pub enum MethodInvocation {
     /// eth_accounts
-    #[serde(rename = "eth_accounts", with = "crate::serde::empty_params")]
+    #[serde(rename = "eth_accounts", with = "edr_eth::serde::empty_params")]
```

Comment:
This coming from `edr_eth` may be a bit weird, but I think it's fine for now. Maybe we could have a separate crate for base utilities like this, but not worth the effort now.